src/result.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_RESULT_H
+#define BITCOIN_RESULT_H
```

Comment:
I think regardless of the namespace, this should be placed in the `util` folder

Reply:
moved 👍🏼